packages/api/src/clickhouse/__tests__/renderChartConfig.test.ts

```diff
@@ -133,7 +133,7 @@ describe('renderChartConfig', () => {
           query_params: query.params,
           format: 'JSON',
         })
-        .then(res => res.json<any>());
+        .then(res => res.json() as any);
```

Comment:
The `make ci-int` command started failing for me this afternoon complaining about how the json method doesn't accept a type param. Not sure why now but this seems to work.

Reply:
I don't think we need to type it here (by default its unknown)